tests/shared/hardware_utils.py

```diff
@@ -47,10 +48,8 @@ def load_hardware_labels(
     gold_dict = get_gold_dict(filename, attribute=attrib)
     cand_total = len(candidates)
     logger.info("Loading {} candidate labels".format(cand_total))
-    pb = ProgressBar(cand_total)
     labels = []
-    for i, c in enumerate(candidates):
-        pb.bar(i)
+    for i, c in enumerate(tqdm(candidates)):
```

Comment:
Note that this order is actually important [[1]](https://github.com/tqdm/tqdm#faq-and-known-issues). The `enumerate` should come around the `tqdm` call.

Reply:
👍